drivers/lora/sx127x.c

```diff
@@ -511,6 +516,17 @@ void SX127xSetRfTxPower(int8_t power)
 	}
 }
 
+uint32_t SX127xGetDio1PinState(void)
+{
+#if SX127X_DIO_GPIO_LEN(0) >= 2
```

Comment:
Is this necessary? Can the SX127x drivers operate without DIO1?
The loramac-node drivers will certainly error out in `SX127xIoIrqInit` if DIO1 doesn't exist.

Reply:
Well, I'm not sure about operating without DIO1, but I've tried to make functional changes as minimal as possible. This is a library update, after all.

Reply:
Yes, but your change to to sx1262x doesn't do any checking that DIO1 exists.
Why does the sx127x driver need it, or why doesn't the sx1262 need it?

Reply:
These drivers are different. For the sx126x driver DIO1 is a hard requirement, the sx127x doesn't require it. I don't think it is good; in my opinion, both drivers need refactoring and more unification. But they work that way at the moment.